proposals/pr-contrib-guide.md

```diff
@@ -0,0 +1,508 @@
+merged. The maintainers do not consistently check on the status of existing pull
+requests and the contribution process stalls. The loss of momentum of the
+contribution process erodes trust with individual contributors, who feel
+ignored, and the community at large, who see stale contributions.
+
+### Goals
+
+Based on the problems identified in the current process, the proposed
+improvements for customer contributions should satisfy the following goals:
+
+1. Maintainers should review community contributions regularly.
+1. Contributors and maintainers should always be able to determine the current
+   state of a contribution and what steps remain to drive the contribution to a
+   resolution.
+1. Maintainers should resolve in-flight contribution reviews before committing
+   to any new reviews.
+
+## Specification
+
+ ![Figure A: Flowchart of contribution guide review
+process.](assets/pr-contribution-guide/contrib-guide-high-level.png "Figure A: Flowchart of contribution
+guide review process.")
+
+**Figure A: Flowchart of contribution guide review process.**
+
+We propose to formalize the existing intake, implementation, and review stages
+to assure they are applied systematically and transparently. In addition, we
+will augment the current process with new stages to clarify what issues are
+available for contribution and what stage they are in. Figure A demonstrates how
+a request and contribution will proceed through the improved process.
+
+We will also publish an improved contributing guide to document the review
+process, acceptance criteria, and expectations for communication. We will use
+existing issue labeling in conjunction with a GitHub Project Board to provide
+transparent status to the community to indicate what is available for
+contribution and to track the progress of a contribution.
+
+The following sections detail the proposed improvements to the existing process.
+
+### Intake stage
+
+A contribution is initiated by opening a GitHub issue. The GitHub issue captures
+information necessary to make the contribution, such as:
+
+1. A description of the requested change.
+1. The number of users in the community that request the change, measured
+   through GitHub :+1: reactions.
+1. Design discussions, including use cases, edge cases, and alternative
+   solutions.
+
+Maintainers will regularly review open GitHub issues and label the ones that are
+ready for community contribution. The decision to label an issue as ready for
+community contribution is ultimately up to the discretion of the maintainers.
+Depending on the issue type (bug or feature request) and area (code or
+documentation), the maintainers will use a set of criteria to decide if the
+issue is ready for contribution. This may include:
+
+1. The change is feasible for a community member to contribute. See [the
+   appendix](#changes-not-available-for-contribution) for details.
+1. Feature requests must have at least 10 :+1: reactions. See the
```

Comment:
Did we want to change this from must to should?

Reply:
yep.